.jenkins/sarus_run_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+set -x
+
+# Set up the compute node environment
+module load daint-gpu
+module add /project/d107/install/modulefiles/
+module load cray-python gcloud sarus
+gcloud auth configure-docker
+
+
+# Set up the working directing for the c48 test
+cd examples
+. /project/d107/install/venv/sn_1.0/bin/activate
+python create_arbitrary_rundir.py c48_config.yml "./c48_test"
+deactivate
+
+cd ${PWD}/c48_test
+cp ../../.jenkins/job_jenkins_sarus .
+export SCRATCH_DIR=${PWD}
+
+# Import the Docker images to be tested on Daint
+dockerfiles=( $( ls ../../docker/Dockerfile.* ) )
+for df in ${dockerfiles[@]}; do
+   arch=${df:24}
+   export FV3_CONTAINER=fv3gfs-compiled:$arch
+   export TAR_FILE=${FV3_CONTAINER/":"/"_"}.tar
+   gsutil copy gs://vcm-ml-public/jenkins-tmp/${TAR_FILE}.gz .
+   gunzip ${TAR_FILE}.gz
+   sarus load ./${TAR_FILE} ${FV3_CONTAINER}
+done
+module unload sarus
+
+# Perform the c48 test for each imported Docker image
+for df in ${dockerfiles[@]}; do
+   export FV3_CONTAINER=fv3gfs-compiled:${df:24}
+   sbatch --wait job_jenkins_sarus
```

Comment:
A failure on one image will prevent the other images from being tested - is this OK?

Reply:
I couldn't think of an easy way to implement concurrent tests without hard-coding the Dockerfile names (which include the various architecture configurations).  I wanted something that would easily pick up a new Dockerfile which one is added (eg. maybe one with OpenMPI instead of MPICH)

Reply:
OK. I think this is sufficient, as long as these are not blocking tests.